docs/docs/building-applications/building-applications.md

```diff
@@ -37,11 +37,14 @@ Groundlight can be used to [apply modern natural-language-based computer vision
 ## Further Reading
 
 For more in-depth guides on various aspects of building applications with Groundlight, check out the following pages:
-
-- [Working with Detectors](working-with-detectors.md): Learn how to create, configure, and use detectors in your Groundlight-powered applications.
-- [Using Groundlight on the edge](edge.md): Discover how to deploy Groundlight in edge computing environments for improved performance and reduced latency.
-- [Handling HTTP errors](handling-errors.md): Understand how to handle and troubleshoot HTTP errors that may occur while using Groundlight.
-
+- **[Grabbing images](1-grabbing-images.md)**: Understand the intricacies of how to submit images from various input sources to Groundlight.
+- **[Working with detectors](2-working-with-detectors.md)**: Learn how to create, configure, and use detectors in your Groundlight-powered applications.
+- **[Confidence levels](3-managing-confidence.md)**: Master how to control the trade-off of latency against accuracy by configuring the desired confidence level for your detectors.
+- **[Handling server errors](4-handling-errors.md)**: Understand how to handle and troubleshoot HTTP errors that may occur while using Groundlight.
+- **[Asynchronous queries](5-async-queries.md)**: Groundlight makes it easy to submit asynchronous queries. Learn how to submit queries asynchronously and retrieve the results later.
+- **[Using Groundlight on the edge](6-edge.md)**: Discover how to deploy Groundlight in edge computing environments for improved performance and reduced latency.
+- **[Industrial applications](7-industrial.md)**: Learn how to apply modern natural-language-based computer vision to your industrial and manufacturing applications.
+  
```

Comment:
I modified this index to include all files in `/building-applications` and bolded the links as I didn't think it was clear that they were links on the website. Also I modified some of the descriptions - please read them over briefly to make sure you agree with the "1 sentence pitches".

Reply:
Also what do you think of the ordering? I tried to make it from least to most power-usery but I couldn't find a place for `Industrial Applications` so I slapped it on the end.